common/chat/conversation.ts

```diff
@@ -91,15 +91,21 @@ const getMatcheeConversation = async (matchees: { studentId: number; pupilId: nu
     return { conversation, conversationId };
 };
 
-async function* getAllConversations(): AsyncIterable<TJConversation> {
+async function* getAllConversations(onlyActive?: boolean): AsyncIterable<TJConversation> {
     assert(TALKJS_SECRET_KEY, `No TalkJS secret key found to get all conversations.`);
     assureChatFeatureActive();
 
+    const filter = { custom: { finished: '!exists' } };
```

Comment:
What if finished: false?

Reply:
We set 
`custom { finished: "course_over" | "match_dissolved"}` if we deactivate chats. So finished can't be false. Or have I misunderstood the question?
https://github.com/corona-school/backend/blob/2f41d00ff9b11814b0bd6e6434122ce4d2caf732/common/chat/deactivation.ts#L58C1-L63C8

But I have forgotten one case: if we reactivate chats. I will add this. @Jonasdoubleyou